src/meta/backup_engine.cpp

```diff
@@ -71,6 +72,12 @@ error_code backup_engine::set_block_service(const std::string &provider)
     return ERR_OK;
 }
 
+void backup_engine::set_backup_path(const std::string &path)
+{
+    ddebug_f("backup path is set to {}.", path);
```

Comment:
Why add a dot in the end?

Reply:
It's just a ending period, sometimes this could be omitted for brevity.